ContactList/ContactList/TableViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  TableViewController.swift
+//  ContactList
+//
+//  Created by Maksym Humeniuk on 5/6/19.
+//  Copyright © 2019 Maksym Humeniuk. All rights reserved.
+//
+
+import UIKit
+
+class TableViewController: UITableViewController {
+    
+    var personsArray = [Person]()
+    
+    @IBOutlet var emptyListView: UIView!
+    
+    @IBOutlet weak var addNewContactButton: UIBarButtonItem!
+    var buttonCopy: UIBarButtonItem?
+    
+    @IBAction func addNewContact(_ sender: Any?) {
+        
+        let controller = self.storyboard!.instantiateViewController(withIdentifier: "ViewControllerForUpdate") as! ViewControllerForUpdate
+        controller.handler = self
```

Comment:
`handler` mostly is being used with closures. `delegate` is mo suitable naming here